feast_trino/trino.py

```diff
@@ -37,6 +38,13 @@ class TrinoOfflineStoreConfig(FeastConfigBaseModel):
     catalog: StrictStr
     """ Catalog of the Trino cluster """
 
+    connector: Dict[str, Any]
+    """
+    Trino connector to use as well as extra parameters.
+    Needs to contain at least the path, for example
+    {"path": "feast_trino.connectors.memory"}
+    """
```

Comment:
This format is interesting to me. It doesn't seem standard. I've seen the normal Pydantic `Field` instances using `description` to provide this help (so if ever the model is converted to jsonschema, the docs contain descriptions too).

Reply:
Should we add a `validator` to ensure the connector has a `path` attribute?